### ISSUE TITLE
Codelense on light theme is less than 4.5:1.

### ISSUE THREAD

**Issue by sanket856** [microsoft/vscode#130397]
GitHubTags:#A11y_VSCode;#A11ySev3;#A11yTCS;#A11yWCAG;#DesktopApp;#Visual Studio Code Client;#ColorContrast;#WCAG1.4.3;#DesktopApp;#Win10;#VisualStudioCode-Win32-Apr2021;#Linked:Bug<<2717>>;

### **Environment details:**

Application Name: Azure Stream Analytics Tools for Visual Studio Code
Windows Version: Win10
Visual studio code version : -1.55.1(User Setup)

### **Repro Steps:**

1. Launch Visual Studio code application with installed Azure Stream Analytics Tools extension .
2. Tab to "Explorer" section in activity bar.
3. Press control +shift +P and edit bar will appear.
4. Type "ASA:Create new project" and press enter to create project.
5. New project will get created.
6. Tab to Input item in Project Tree Section
7. Right click on input and navigate to "ASA : Add Input" using arrow key and press enter.
8. Select IoT hub from available option using enter key.
9. Select "select from azure subscriptions" from available option using enter key.
10. Type any name for input file and press enter to create Input json file.
11. Tab to "Select Subscription" button.
12. Start AI4D tool and check the color contrast ratio of select subscription text.

###  **Actual Result:**
The color contrast ratio of the text "Select Subscription" button available in input json file is less than 4.5:1.

Note: - This issue is also observed with "Select IoT Hub", "Select Shared Access Policy name" text

### **Expected Result:**

The color contrast ratio of the text "Select Subscription" button available in input json file should be greater than or equal to 4.5:1.

### **User Impact:**

The low vision users may not able to see the "Select Subscription" text available in the input json file.

**Comment by misolori** [comment]
Can you please upgrade your VS Code to the latest version, we just launched 1.59 and you are on 1.55, as we fixed this in a previous version. 

See https://github.com/microsoft/vscode/issues/126899 for the duplicate issue that was fixed.

**Comment by sanket856** [comment]
@misolori I have checked the color contrast in latest version ,ratio is 3.151:1. The color contrast ratio of the text should be greater than or equal to 4.5:1.
